frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/block_action_add_button.dart

```diff
@@ -21,25 +24,36 @@ class BlockAddButton extends StatelessWidget {
   Widget build(BuildContext context) {
     return BlockActionButton(
       svgName: 'editor/add',
-      richMessage: const TextSpan(
+      richMessage: TextSpan(
         children: [
           TextSpan(
-            // todo: l10n.
-            text: 'Click to add below',
+            text: LocaleKeys.blockActions_addBelowTooltip.tr(),
+          ),
+          const TextSpan(text: '\n'),
+          TextSpan(
+            text: LocaleKeys.blockActions_addAboveTooltip.tr(),
           ),
         ],
       ),
       onTap: () {
+        final isAltPressed =
+            KeysPressedManager.of(context)?.isAltPressed ?? false;
```

Comment:
Is there any reason for not using `RawKeyboard.instance.keysPressed` instead?

The `KeysPressedManager` can do these calculation.

- only contains alt, shift, cmd, ...
- contains alt, shift, cmd, ...